CHANGELOG.md

```diff
@@ -9,6 +9,51 @@ The ManageIQ organization is continuously adding new smaller repositories.  The
 - [manageiq-providers-vmware CHANGELOG](https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/CHANGELOG.md)
 - [manageiq-ui-classic CHANGELOG](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/CHANGELOG.md)
 
+## Sprint 67 ending 2017-08-21
```

Comment:
Actually,..I was confused on the "unreleased" heading so I asked Jason,....he said it doesn't apply to this repo.